src/tests/utests/xi_utest_data_desc.c

```diff
@@ -244,6 +244,259 @@ XI_TT_TESTCASE_WITH_SETUP(
         tt_fail();
     } )
 
+#ifdef XI_MEMORY_LIMITER_ENABLED
```

Comment:
I've checked this condition. Unfortunately I did not find better solution. The only tool in our hands to drive these `*make_desc*` functions to the test scenario is the memory limiter. But at least we have this tool. And congrats great trick with this tool to use it for this purpose.

Reply:
Yes, this doesn't feel great to me, either, but ... here we are. :)